storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/common.cpp

```diff
@@ -364,78 +345,38 @@ RS_Status SetOperationPKCol(const NdbDictionary::Column *col, NdbOperation *oper
     if (request->PKValueNDBStr(colIdx, col, &charStr) != 0) {
       return RS_CLIENT_ERROR(ERROR_019);
     }
+
     if (operation->equal(request->PKName(colIdx), charStr, data_len) != 0) {
```

Comment:
colIdx, is this a local colIdx for the query, or is it the colIdx of the column in the table, if so, there is an equal method using colIdx directly which makes it a lot faster